internal/kptfile/pkgfile.go

```diff
@@ -121,18 +121,41 @@ func (updatedKf *KptFile) MergeOpenAPI(localKf, originalKf KptFile) error {
 	}
 	oriDef := original.Field("definitions")
 
-	err = localDef.Value.VisitFields(func(node *yaml.MapNode) error {
-		key := node.Key.YNode().Value
-		if shouldSkipCopy(updatedDef, localDef, oriDef, key) {
-			return nil
+	localKeys, _ := localDef.Value.Fields()
+	updatedKeys, _ := updatedDef.Value.Fields()
+    keys := append(updatedKeys, localKeys...)
+
+	unique := make(map[string]bool, len(keys))
+	for i:= 0; i<len(keys); i++ {
```

Comment:
for _, key : range keys is more go like.

Reply:
Done.